scripts/src/metrics.ts

```diff
@@ -45,9 +45,9 @@ export function offersReturned(numOffers: number, appId: string) {
 	statsd.histogram("offers_returned", numOffers, undefined, { app_id: appId });
 }
 
-export function reportClientError(error: MarketplaceError, headers: { [name: string]: string }) {
-	const data = Object.assign({ status: error.status.toString(), title: error.title, code: error.code }, headers);
-	statsd.increment("client_error", 1, undefined, data);
+export function reportClientError(error: MarketplaceError, appId: string) {
+	statsd.increment("client_error", 1, undefined,
```

Comment:
Why `undefined` and not `null`? null signifies no value, undefined means this argument wasn't processed/touched by the code, which is indicative of a bug.